lib/Controller/ApiController.php

```diff
@@ -167,15 +196,51 @@ public function export(array $migrators): DataResponse {
 	 * @NoSubAdminRequired
 	 * @PasswordConfirmationRequired
 	 */
-	public function import(string $path): DataResponse {
-		$user = $this->userSession->getUser();
+	public function import(string $path, string $targetUserId): DataResponse {
+		$sourceUser = $this->userSession->getUser();
```

Comment:
So source user is actually author of the import action, I would rename the field to author to avoid confusion